java/ql/src/Security/CWE/CWE-925/Good.java

```diff
@@ -0,0 +1,16 @@
+...
+IntentFilter filter = new IntentFilter(Intent.ACTION_SHUTDOWN);
+BroadcastReceiver sReceiver = new ShutDownReceiver();
+context.registerReceiver(sReceiver, filter);
+...
```

Comment:
I'd suggest making this look more like valid Java code:

```suggestion
// ...
IntentFilter filter = new IntentFilter(Intent.ACTION_SHUTDOWN);
BroadcastReceiver sReceiver = new ShutDownReceiver();
context.registerReceiver(sReceiver, filter);
// ...
```